pyanaconda/errors.py

```diff
@@ -119,20 +113,6 @@ def _storage_reset_handler(self, exn):
         else:
             return ERROR_RAISE
 
-    def _invalidImageSizeHandler(self, exn):
-        message = _("The ISO image %s has a size which is not "
-                    "a multiple of 2048 bytes.  This may mean "
-                    "it was corrupted on transfer to this computer."
-                    "\n\n"
-                    "It is recommended that you exit and abort your "
-                    "installation, but you can choose to continue if "
-                    "you think this is in error. Would you like to "
-                    "continue using this image?") % exn.filename
```

Comment:
That would be great, but I don't think that it is a good idea to store unused strings in our code base. It is not likely that we will need the same strings in the future. Also, do we want people to translate strings that we keep just in case?

Reply:
At least for me, the real issue is that we will now have "generic error" instead of useful details. Keeping the strings is meaningless if they are not used.

Reply:
I have checked to code and there is no easy way how to propagate this error to the user. We cannot really keep this error handler, because it won't work after the modularization. A DBus task cannot and shouldn't wait for a user input. I have moved part of the error message to the warning log and I can create an issue for improving the user experience, but that is out of scope of this PR.